mock/src/main/java/com/tngtech/keycloakmock/impl/handler/TokenIntrospectionRoute.java

```diff
@@ -0,0 +1,83 @@
+package com.tngtech.keycloakmock.impl.handler;
+
+import com.tngtech.keycloakmock.impl.helper.TokenHelper;
+import io.jsonwebtoken.Claims;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.vertx.core.Handler;
+import io.vertx.core.json.Json;
+import io.vertx.ext.web.RoutingContext;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+public class TokenIntrospectionRoute implements Handler<RoutingContext> {
```

Comment:
Please add at least one happy test case to verify that passing in a valid token will return its claims plus the "active" field set to true. Also, please add some tests for the error cases (expired token, missing token, premature token, malformed token).